src/sharepoint/rest/lists.ts

```diff
@@ -8,17 +8,21 @@ import { Queryable, QueryableCollection } from "./queryable";
 import { QueryableSecurable } from "./QueryableSecurable";
 import * as Util from "../../utils/util";
 import { TypedHash } from "../../collections/collections";
-import * as Types from "./types";
+
+import { ChangeQuery } from "../../types/changequery";
```

Comment:
One of the things I wanted to avoid with this change was the need to import each type individually. Is there a way to combine them all into a central file we can include once? That also gives us Types.* intellisense once it is included.